### PR TITLE
Blog post for lab meeting 22 August 2019

### PR DESCRIPTION
**Pull request by KirstieJane**
Please review/comment asap

### CODE REVIEW

**Review by sgibson91 — APPROVED** [review]
💖

File: _posts/blog/2019-08-22-2019-08-22-lab-meeting.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+She's had lots of useful conversations at the Turing, and we had some great discussion at the end of lab meeting.
+This is going to be an awesome paper that she's currently writing!
+
+**Georgia** is working out comms strategies for the [citizen science project](https://github.com/alan-turing-institute/AutisticaCitizenScience/).
+She'd love suggestions how to engage and involve people? 
+
+**Elizabeth** is brainstorming what an ideal Jupyter "authoring experience" could look like.
+She'd love comments on [this initial sketch of a workflow](https://github.com/jupyter/jupyter-book/issues/236)!
+
+**Ang** wonders if there's a better way to test the heterogenetity of one-dimensional continuous data across two groups?
```
Comment (sparkler0323) [inline _posts/blog/2019-08-22-2019-08-22-lab-meeting.md]:
```suggestion
**Ang** wonders if there's a better way to test the heterogeneity of one-dimensional continuous data across two groups?
```

**Review by sparkler0323 — APPROVED** [review]
👍

File: _posts/blog/2019-08-22-2019-08-22-lab-meeting.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+The linked blog is great discussion of the pros and cons of rainbow colourmaps 🌈
+
+**Yini** is "Learning Python the Hard Way" (via [the book](https://smile.amazon.co.uk/Learn-Python-Hard-Way-Introduction/dp/0321884914/ref=smi_www_rco2_go_smi_g8682124849?_encoding=UTF8&%2AVersion%2A=1&%2Aentries%2A=0&ie=UTF8) by Zed A Shaw recommended by Sarah and Georgia.
+She found this video on youtube really helpful: [https://www.youtube.com/watch?v=9k8s8r5DXNU](https://www.youtube.com/watch?v=9k8s8r5DXNU).
+
+Yini also recommended the "Tell me a story" point of view article by Joshua Sanes at eLife (doi: [10.7554/eLife.50527](https://doi.org/10.7554/eLife.50527)).
+Sanes suggests starting with the introduction and results sections of a paper and leaving the figures to later.
+
+(Side note: This is not Kirstie's style at all, she's super into figures to tell the story, but its worth reading and seeing what fits your workflow best!)
+
```
Comment (minnieho1115) [inline _posts/blog/2019-08-22-2019-08-22-lab-meeting.md:50]: It is really coooool and logical to discribe a research only using 1 or 2 figures!

**Review by minnieho1115 — APPROVED** [review]
I can always learn a lot from the lab meetings. Thanks a lot!:smirk:

**Comment by ajrcampbell** [comment]
👍

**Review by emdupre — APPROVED** [review]

**Pull request merged**